master/buildbot/config.py

```diff
@@ -339,6 +341,9 @@ def copy_str_param(name, alt_key=None):
         if 'slave_debug_url' in config_dict:
             self.slave_debug_url = config_dict["slave_debug_url"]
 
+        if 'slaveManagerUrl' in config_dict:
```

Comment:
could you check that we are correctly handle the case of setting this value to None ?